sycl/CMakeLists.txt

```diff
@@ -184,6 +184,21 @@ install(DIRECTORY ${OPENCL_INCLUDE}/CL
   COMPONENT opencl-headers
 )
 
+if (NOT TARGET LLVMGenXIntrinsics)
+  message(STATUS "vc-intrinsics are missing. Will try to download them from github.com")
+  include(FetchContent)
+  FetchContent_Declare(vc-intrinsics
+    GIT_REPOSITORY https://github.com/intel/vc-intrinsics.git
+    GIT_TAG        cce6e48c28eb850d7dadd30841c0d95f009bbca1
```

Comment:
Do you think using origin/master and most up-to-date version instead of using this hash tag is risky now?

Reply:
Thanks, this is a good question to discuss.
As for now, we do not have strong testing when updating vc-intrinsics. I think it is better to use a fixed revision so that vc-intrinsics update does not accidentally break the build for developers.

Reply:
@v-klochkov, we need to use a tag or a hash which is known (and tested) to work with the product to minimize risks of regression due to vc-intrinsic code changes.